File: eventplatformUI/src/app/landingpage/domains/Timing.ts
import { Category } from "./Category";

​
export class Timing {
  showId:string;
  showTime:string;
  categories:Category[];
​
  public Timing(showTime:string, showId:string, categories:Category[]) {
     this.showTime = showTime;
     this.showId = showId;
     this.categories = categories;
  }
}

File: show-service/src/main/java/com/stackroute/domain/Timing.java
package com.stackroute.domain;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Timing {
     @Id
     private String showId;
     private String showTime;

     private List<Category> categories;
}
